package.json

```diff
@@ -71,10 +71,10 @@
     "@types/node": "^12.20.50",
     "@typescript-eslint/eslint-plugin": "^5.27.1",
     "@typescript-eslint/parser": "^5.27.1",
-    "@venusprotocol/governance-contracts": "2.0.0",
+    "@venusprotocol/governance-contracts": "2.1.0",
     "@venusprotocol/oracle": "2.0.0",
     "@venusprotocol/protocol-reserve": "2.1.0",
-    "@venusprotocol/venus-protocol": "8.0.0",
+    "@venusprotocol/venus-protocol": "9.0.0-dev.9",
```

Comment:
Let's wait for https://github.com/VenusProtocol/venus-protocol/pull/488, and set `9.0.0`

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/405/commits/be3c91818daa1f6952115f7aa8be52a99ad0045f